cranelift/filetests/filetests/isa/x86/simd-vselect-run.clif

```diff
@@ -0,0 +1,59 @@
+test run
+set enable_simd
+target x86_64 haswell
+
+function %vselect_i8x16() -> b1 {
+block0:
+    v1 = vconst.b8x16 [false true false true false true true true true true false false false false false false]
+    v2 = vconst.i8x16 [100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115]
+    v3 = vconst.i8x16 [200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215]
+    v4 = vselect v1, v2, v3
+
+    v5 = vconst.i8x16 [200 101 202 103 204 105 106 107 108 109 210 211 212 213 214 215]
+    v6 = icmp eq v4, v5
+    v7 = vall_true v6
+    return v7
+}
+; run
```

Comment:
This and below can now use better syntax:
```suggestion
    v4 = vselect v1, v2, v3
    return v4
}
; run: %vselect_i8x16() == [200 101 202 103 204 105 106 107 108 109 210 211 212 213 214 215]
```

And, even better, could pass in the vectors in the run statement:
```
; run: %vselect_i8x16([...], [...], [...]) == [...]